src/Views/Legal/Accessibility.cshtml

```diff
@@ -0,0 +1,39 @@
+@{ ViewBag.Title = $"Find teacher training courses"; }
+
+<main class="govuk-main-wrapper" id="main-content" role="main">
+  <div class="govuk-width-container">
+    <h1 class="govuk-heading-xl">Accessibility statement for Find teacher training courses</h1>
```

Comment:
All this content needs to be in a 2/3rds container to stop line length being too long.